.github/workflows/dependencies.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update dependencies
+on:
+  schedule:
+    - cron: '0 3 * * 6'  # Run every saturday at 3 am.
+jobs:
+  update_dependencies:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Cache pip dir
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('requirements.txt') }}
+      - name: Update dependencies
+        run: | 
+          python -m pip install --upgrade pip setuptools poetry
+          python -m pip install -r lib/galaxy/dependencies/lint-requirements.txt
+          python -m pip freeze -l > lib/galaxy/dependencies/pinned-lint-requirements.txt
+          poetry update -v --lock
+          poetry export -f requirements.txt --without-hashes --output lib/galaxy/dependencies/pinned-requirements.txt
+          poetry export --dev -f requirements.txt --without-hashes --output lib/galaxy/dependencies/dev-requirements.txt
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: Update dependencies
+          title: Update dependencies
+          body: Run make update-dependencies
```

Comment:
```suggestion
          commit-message: Update Python dependencies
          title: Update Python dependencies
          body: using `make update-dependencies`.
```

Reply:
I'd rather not split a sentence across title and body. I think the original body describes precisely what was done.

Reply:
Fair enough, maybe just add the backticks?